### PR TITLE
Fix spelling errors and shorten some code.

### PR DESCRIPTION
**Pull request by sky-2002**

### CODE REVIEW

**Comment by bobvanluijt** [comment]
@byronvoorbach and/or @StefanBogdan – can one of you take a look at this PR, please?

File: weaviate/util.py [hunk 1]
```diff
@@ -198,13 +196,13 @@ def is_object_url(url: str) -> bool:
...
     ----------
-    input : str
+    url : str
         The URL to be validated.
 
     Returns
     -------
     bool
-        True if the 'input' is a valid path to an object.
+        True if the 'input' ie the 'url' is a valid path to an object.
```
Comment (StefanBogdan) [inline weaviate/util.py:205]: I think the fixed line should have been this one:
```
 True if the 'url' is a valid path to an object.
```

File: weaviate/data/crud_data.py [hunk 2]
```diff
@@ -248,11 +248,11 @@ def update(self,
...
-            data_object: Union[dict, str],
-            class_name: str,
-            uuid: Union[str, uuid_lib.UUID],
-            vector: Sequence=None
-        ) -> None:
+                data_object: Union[dict, str],
+                class_name: str,
+                uuid: Union[str, uuid_lib.UUID],
+                vector: Sequence = None
+                ) -> None:
```
Comment (StefanBogdan) [inline weaviate/data/crud_data.py:255]: I do not like this indentation, I already used 2 tabs, this change now is 3 tabs, which is one too many. 

Also when an argument has a default value, there are no spaces around the `=`, only when a variable is assigned (this is the style I use).

File: weaviate/data/crud_data.py [hunk 3]
```diff
@@ -346,10 +346,10 @@ def replace(self,
...
 
     def get_by_id(self,
-            uuid: Union[str, uuid_lib.UUID],
-            additional_properties: List[str]=None,
-            with_vector: bool=False
-        ) -> Optional[dict]:
+                  uuid: Union[str, uuid_lib.UUID],
+                  additional_properties: List[str] = None,
+                  with_vector: bool = False
+                  ) -> Optional[dict]:
```
Comment (StefanBogdan) [inline weaviate/data/crud_data.py:352]: Same as the other ones above.

File: weaviate/data/crud_data.py [hunk 4]
```diff
@@ -569,11 +569,11 @@ def exists(self, uuid: Union[str, uuid_lib.UUID]) -> bool:
...
-            data_object: Union[dict, str],
-            class_name: str,
-            uuid: Union[str, uuid_lib.UUID, None]=None,
-            vector: Sequence=None
-        ) -> dict:
+                 data_object: Union[dict, str],
+                 class_name: str,
+                 uuid: Union[str, uuid_lib.UUID, None] = None,
+                 vector: Sequence = None
+                 ) -> dict:
```
Comment (StefanBogdan) [inline weaviate/data/crud_data.py:576]: Same as the other ones above.

File: weaviate/data/crud_data.py [hunk 5]
```diff
@@ -700,7 +700,7 @@ def _get_params(additional_properties: Optional[List[str]], with_vector: bool) -
     if additional_properties:
         if not isinstance(additional_properties, list):
             raise TypeError("Additional properties must be of type list "
-                f"but are {type(additional_properties)}")
+                            f"but are {type(additional_properties)}")
```
Comment (StefanBogdan) [inline weaviate/data/crud_data.py:703]: I do not see the need for this many tabs, it does not make it more readable. I would suggest doing this instead:
```python
raise TypeError(
    "Additional properties must be of type list "
    f"but are {type(additional_properties)}"
)
```
or, if line not longer than 100 characters:
```python
raise TypeError(
    f"Additional properties must be of type list but are {type(additional_properties)}"
)
```

File: weaviate/util.py [hunk 6]
```diff
@@ -48,7 +47,7 @@ def image_encoder_b64(image_or_image_path: Union[str, BufferedReader]) -> str:
         content = image_or_image_path.read()
     else:
         raise TypeError('"image_or_image_path" should be a image path or a binary read file'
-            ' (io.BufferedReader)')
+                        ' (io.BufferedReader)')
```
Comment (StefanBogdan) [inline weaviate/util.py:50]: I would do it like this now:
```python
raise TypeError(
    '"image_or_image_path" should be a image path or a binary read file '
    '(io.BufferedReader)'
)
```
or, if line not longer than 100 charachters:
```python
raise TypeError(
    '"image_or_image_path" should be a image path or a binary read file (io.BufferedReader)'
)
```

**Review by StefanBogdan — CHANGES_REQUESTED** [review]

File: weaviate/data/crud_data.py [hunk 7]
```diff
@@ -43,24 +43,24 @@ def __init__(self, connection: Connection):
...
-            data_object: Union[dict, str],
-            class_name: str,
-            uuid: Union[str, uuid_lib.UUID, None]=None,
-            vector: Sequence=None
-        ) -> str:
+               data_object: Union[dict, str],
+               class_name: str,
+               uuid: Union[str, uuid_lib.UUID, None] = None,
+               vector: Sequence = None
+               ) -> str:
```
[the remark this answers is not in the document]
Comment (sky-2002) [reply weaviate/data/crud_data.py:50]: Ohk, I will keep this in mind 👍, and fix that

File: weaviate/data/crud_data.py (hunk 5 above)
Comment (sky-2002) [reply weaviate/data/crud_data.py:703]: Ok 👍

**Comment by sky-2002** [comment]
Hello @StefanBogdan , I have made changes as per review. Let me know if its fine or any changes to be made.

**Pull request merged**